adhocracy4/comments_async/static/comments_async/comment.jsx

```diff
@@ -201,13 +200,21 @@ export default class Comment extends React.Component {
       }
       comment = (
         <div className={'a4-comments__text' + (this.state.anchored ? ' a4-comments__text--highlighted' : '')}>
-          <ReactMarkdown
-            disallowedElements={['h1', 'h2', 'h3', 'h4', 'input', 'table', 'thead', 'tr', 'th']}
-            unwrapDisallowed
-            plugins={[[gfm, { singleTilde: false }]]}
-          >
-            {content}
-          </ReactMarkdown>
+          {this.props.is_moderator_marked
+            ? <mark>
```

Comment:
this seems to highlight/mark the entire line in cases where the comment is shorter. Is this what we want?

Reply:
nope but really not sure how else to do it with the markdown, did have a play yesterday but it's not a big story, also I imagine a moderator marked comment that is amazing probable will be longer then 2-3 words hopefully

Reply:
it's not ideal I know!